static/js/services/translation.js

```diff
@@ -30,25 +30,25 @@ define(["jed", "underscore"], function(Jed, _) {
 		this._ = _.bind(function() {
 			if (domain && context) {
 				return _.bind(function(singular) {
-					var vars = Array.prototype.slice.call(arguments, 0);
+					var vars = Array.prototype.slice.call(arguments, 1);
```

Comment:
Why has this been changed in the first place? It is like this in current release (master).